app/models/foreman_docker/docker.rb

```diff
@@ -131,13 +131,16 @@ def bootstrap(args)
     end
 
     def client
-      @client ||= ::Fog::Compute.new(
-          :provider         => 'fogdocker',
-          :docker_username  => user,
-          :docker_password  => password,
-          :docker_email     => email,
-          :docker_url       => url
-      )
+      @client ||= begin
+                    opts = {
+                      :provider         => 'fogdocker',
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.